nekoyume/Assets/_Scripts/UI/ChatPopup.cs

```diff
@@ -50,5 +51,11 @@ public override void Show(bool ignoreShowAnimation = false)
         {
             base.Show(true);
         }
+
+        public override void Close(bool ignoreCloseAnimation = false)
+        {
+            base.Close(ignoreCloseAnimation);
+            AudioController.PlayClick();
```

Comment:
It is recommended to place this line in the callback lambda method of `confirm.onClick`.